### ISSUE TITLE
Results data structure error

### ISSUE THREAD

**Issue by JRiad** [strusoft/femdesign-api#576]
I input a list of three separate fem design models into the "run analysis" component I would expect to see my results (point support reactions in this case) in three different branches. All the results are there, but they are all in one long list (index 0), and not a datatree with three branches (0),(1) and (2) for each model
![image](https://user-images.githubusercontent.com/89396751/199202515-21c54a81-2809-43a0-a6d6-6057de20b806.png)

**Comment by xRadne** [comment]
@Marco-Pellegrino See, I told you someone would plug several models in to the RunAnalysis or ReadStr components ;)

**Comment by Marco-Pellegrino** [comment]
True. It has not fully implemented as really few user will need the feature.
We will fix that. 

As a workaround, I suggest you to simply split the output list as you know the number of elements to performe a new list.

**Comment by JRiad** [comment]
@Marco-Pellegrino Sure, it is an easy workaround, no rush 👍 The issue originally comes from a user question, so we have at least one guy out there using this workflow 😉
